gap/FinSetsForCAP.gi

```diff
@@ -954,6 +982,15 @@ end );
 ##
 Finalize( FinSets );
 
+##
+InstallMethod( ViewObj,
```

Comment:
Why do you introduce this hardcoded `ViewObj` method?

Reply:
We just talked about this: we do not want to print `<A projective object in FinSets>` since this does not contain any additional information.